include/identity.php

```diff
@@ -663,6 +668,10 @@ function advanced_profile(App $a) {
 			$profile['age'] = array( t('Age:'), $age );
 		}
 
+		if ($age = age($a->profile['dob'], $a->profile['timezone'], '')) {
```

Comment:
Against what version of the code are you rebasing? See #3573  for the current code at the point, it still includes this statement, but also a check for not setting a age from the NULL_DATE.

@annando age() calculates the age but honouring the potential timezone shift I think..